lib/presentation/util/helper/responsive_widget.dart

```diff
@@ -36,8 +36,10 @@ class ResponsiveWidget extends StatelessWidget {
   final Widget mediumScreen;
   final Widget smallScreen;
 
-  static final int minLargeWidth = 960;
-  static final int minMediumWidth = 640;
+  static final int minLargeWidth = 950;
+  static final int minMediumWidth = 600;
```

Comment:
why we need to change this?

Reply:
These are the two most common values for large and xlarge tablet screens. You can see more here. [https://developer.android.com/training/multiscreen/screensizes](https://developer.android.com/training/multiscreen/screensizes)

Reply:
what is the unit for this width? (pixel? ...)

Reply:
and IMO 950 is not a common value

Reply:
it is dp

Reply:
I have read through the documentation it is a popular desktop value. I don't remember the link of that document. Maybe we will discuss this value again.

Reply:
iOS using dp too?

Reply:
In flutter we use the same px. If dp is large then dp ~ px